box-content-sdk/src/main/java/com/box/androidsdk/content/models/BoxFolder.java

```diff
@@ -101,6 +102,34 @@ public static BoxFolder createFromId(String folderId) {
     }
 
     /**
+     * A convenience method that copies all properties of a folder except the item collection.
+     *
+     * @param folder Original folder to create a copy off of
+     * @return copy of a folder with an empty item collection
+     */
+    public static BoxFolder copyFolderWithNoItems(final BoxFolder folder) {
```

Comment:
This still feels a bit more like a utility method than something that applies to the folder model. Any issues with this being moved to SdkUtils instead?